tests/test_spatial_filters.py

```diff
@@ -10,6 +11,20 @@ def ring_as_wkt(*points):
     return "(" + ",".join(f"{x} {y}" for x, y in points) + ")"
 
 
+def bbox_as_wkt_polygon(min_x, max_x, min_y, max_y):
```

Comment:
Not done - this particular piece of code is a test helper that is only used in this file, so, as long as it is a useful test helper, it can have its arguments in any reasonable order. Right now none of the tests test anti-meridian crossing spatial filters so it doesn't make any difference.
The actual production code does need to actually deal with CRS's that cross the anti-meridian, at some point, but this is still on the list of TODOs. When it does, I think we will still have some (min-x, max-x, min-y, max-y) though, since that is the official GPKG spec, and we store our geometries according to that spec. In any case, not dealt with in this PR.

Reply:
> some (min-x, max-x, min-y, max-y) though, since that is the official GPKG spec, and we store our geometries according to that spec.

Potentially makes sense for a single geometry in some situations, falls apart for anything else. Add a comment that a CRS-aware bounds method should be used instead of this unless absolutely necessary.